source/app/scss/base/_lists.scss

```diff
@@ -1,7 +1,6 @@
 ul,
 ol {
-  list-style-type: none;
-  margin: 0;
+  margin: 0 0 0 25px;
```

Comment:
I tested it and have some bugs :-(

![image](https://cloud.githubusercontent.com/assets/11995337/24589386/143257f2-17e2-11e7-98eb-a62703177891.png)